feature-libs/product/bulk-pricing/components/bulk-pricing-table/bulk-pricing-table.component.spec.ts

```diff
@@ -21,4 +54,47 @@ describe('BulkPricingTableComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('formatQuantity', () => {
+    const mockTierWithMaxQuantity: BulkPrice = {
+      minQuantity : 1,
+      maxQuantity : 5,
+      value: 200,
+      discount: 20
+    };
+
+    const mockTierWithoutMaxQuantity: BulkPrice = {
+      minQuantity : 1,
+      value: 200,
+      discount: 20
+    };
```

Comment:
For sake of consistency with other suites, can we move them up, to the top of the file?